CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 ### Features
 
+- Add RNSentrySDK APIs support to @sentry/react-native/expo plugin ([#4633](https://github.com/getsentry/sentry-react-native/pull/4633))
```

Comment:
Nit. It would be nice to include an example code snippet and a small summary of what will the flag do.